src/Symfony/Component/Console/Application.php

```diff
@@ -466,7 +466,9 @@ public function get($name)
             $command = $this->commands[$name];
         } elseif ($this->commandLoader && $this->commandLoader->has($name)) {
             $command = $this->commandLoader->get($name);
-            $command->setName($name);
+            if (!$command->getName()) {
```

Comment:
> I'm even asking myself if we shouldn't throw if !$command->getName() rather than letting the command being named by an alias and have some WTF moments

Tend to agree, that's what `add()` actually does :)